Framework/Algorithms/src/CreateUserDefinedBackground.cpp

```diff
@@ -187,6 +180,61 @@ CreateUserDefinedBackground::createBackgroundWorkspace(
 }
 
 /**
+ * Get Y storage mode for background data.
+ *
+ * - Point data -> counts
+ * - Histogram and distribution -> frequencies
+ * - Histogram, not distribution -> counts, BUT if the data was plotted using
+ * the "normalize histogram to bin width" option then it needs to be multiplied
+ * by bin width
+ *
+ * @param data :: [input] Input workspace
+ * @param yBackground :: [input, output] Y background data - could be changed
+ * depending on options
+ * @returns :: Y storage mode to be used
+ */
+HistogramData::Histogram::YMode CreateUserDefinedBackground::getBackgroundYMode(
+    const API::MatrixWorkspace_const_sptr &data,
+    std::vector<double> &yBackground) const {
+  if (data->isHistogramData()) {
+    if (data->isDistribution()) {
+      return Histogram::YMode::Frequencies;
+    } else {
+      if ("On" ==
+          Kernel::ConfigService::Instance().getString(AUTODISTRIBUTIONKEY)) {
+        // User has selected the option "normalize histogram to bin width"
+        multiplyByBinWidth(data, yBackground);
+      }
+      return Histogram::YMode::Counts;
+    }
+  } else {
+    return Histogram::YMode::Counts;
+  }
+}
+
+/**
+ * Multiplies the Y background data by input workspace bin width, to convert
+ * from frequencies to counts
+ * @param data :: [input] Input workspace - must be histogram data
+ * @param yBackground :: [input, output] Y background data - will be multiplied
+ * by bin width
+ */
+void CreateUserDefinedBackground::multiplyByBinWidth(
+    const API::MatrixWorkspace_const_sptr &data,
+    std::vector<double> &yBackground) const {
+  if (!data->isHistogramData()) {
+    throw std::invalid_argument(
+        "Data is not histogram data: cannot multiply background by bin width");
+  }
+
+  const auto &X = data->binEdges(0);
+  for (size_t i = 0; i < yBackground.size(); ++i) {
+    const double width = X[i + 1] - X[i];
+    yBackground[i] *= width;
```

Comment:
The errors of the background spectrum are set to zero so don't need to be multiplied. (The 'real' error would depend on how accurate the user was with their clicking on the data points)